network.go

```diff
@@ -307,6 +533,35 @@ func bridgeNetworkPair(netPair *NetworkInterfacePair) error {
 	return nil
 }
 
+func untapNetworkPair(netPair NetworkInterfacePair) error {
```

Comment:
Ditto for this function, I think we can factorize a lot with `unBridgeNetworkPair()`.